src/mlpack/tests/binarize_test.cpp

```diff
@@ -46,9 +46,9 @@ TEST_CASE("BinarizeOneDimension", "[BinarizeTest]")
 TEST_CASE("BinerizeAll", "[BinarizeTest]")
 {
   mat input;
-  input << 1 << 2 << 3 << endr
-        << 4 << 5 << 6 << endr // this row will be tested
-        << 7 << 8 << 9;
+  input = { { 1, 2, 3 },
+            { 4, 5, 6 }, // this row will be tested
```

Comment:
```suggestion
            { 4, 5, 6 }, // This row will be tested.
```

I know the comment was there before, but we should be consistent and use the same comment style.